dev/src/bundle.ts

```diff
@@ -109,10 +85,11 @@ export class BundleBuilder {
   private addBundledDocument(snap: DocumentSnapshot) {
     const docProto = snap.toDocumentProto();
     this.documents.set(snap.id, {
-      document: docProto,
+      document: snap.exists ? docProto : undefined,
```

Comment:
Nit: I don't know that snap.toDocumentProto() does if the document doesn't exist. I would not need to check if you called it here inline.

Reply:
there is another `docProto.name` below, so I prefer to keeping it this way.